### DISCUSSION TITLE
How to set be or fe conf JAVA_OPTS(-Xmx xG) size?

### DISCUSSION THREAD

**Discussion by dzygoon** [apache/doris#29780] [A - General / Q&A] [answered]
I depoly an cluster,4 fe node and 3 be node,I wonder what Xms should be set size to in the JAVA_OPTS configuration?
Each of my machines has 64GB of RAM.
thanks.

**Comment by zy-kkk** [comment, marked as answer]
If you have only one service on each machine and there is no mixing of FE and BE, then the JVM memory of FE can be adjusted to 16G-32G, and the JVM memory of BE can be adjusted to 1G-4G.

**Reply by dzygoon** [reply]
thanks for your answer.Yes.The nodes of each of my machines are independent.Be node had 64G ram,just set 4G?
